app/helpers/occupation_standards_helper.rb

```diff
@@ -2,4 +2,10 @@ module OccupationStandardsHelper
   def ojt_type_display(occupation_standard)
     occupation_standard.ojt_type&.titleize
   end
+
+  def filters_class
+    unless params[:state_id].present?
+      "hidden"
+    end
+  end
```

Comment:
I think we should probably put all the filter params within a `filter:` hash, since adding each individual filter here is going to be somewhat annoying. But I figured it could wait until the next filter that we add.